### ISSUE TITLE
PII is hidden error

### ISSUE THREAD

**Issue by newbieDev01** [Azure-Samples/active-directory-dotnet-webapp-groupclaims#51]
Hi ,
I cloned this repository in an attempt to find out how to access the Azure AD groups. When I Run the application and click on task, I am getting the below error,
![image](https://user-images.githubusercontent.com/47981290/53343947-6bf1ac00-3909-11e9-9bb0-b8eb53f7ca2e.png)

Can anybody help?

**Comment by jmprieur** [comment]
Did you follow the instructions in the README.md : https://github.com/Azure-Samples/active-directory-dotnet-webapp-groupclaims#step-2--register-the-sample-with-your-azure-active-directory-tenant

To enable PII, you need to add a line 

```CSharp
// Since IdentityModel version 5.2.1 (or since Microsoft.AspNetCore.Authentication.JwtBearer version 2.2.0),
// PII hiding in log files is enabled by default for GDPR concerns.
// For debugging/development purposes, one can enable additional detail in exceptions by setting IdentityModelEventSource.ShowPII to true.
Microsoft.IdentityModel.Logging.IdentityModelEventSource.ShowPII = true;
```

**Comment by jmprieur** [comment]
Leaving it opened as I want to document this.

**Comment by robwillup** [comment]
I was getting the same error described above.
I managed to resolved this by concatenating "v2.0" to the end of the "Authority" string as described [here](https://stackoverflow.com/questions/43276977/azure-ad-b2c-error-idx10501-signature-validation-failed).

**Comment by TiagoBrenck** [comment]
Closing as we are archiving this sample. 

The newer one can be found at: https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/tree/master/5-WebApp-AuthZ/5-2-Groups
